app/layout/index.styl

```diff
@@ -23,7 +23,7 @@
     z-index: 2
 
   &__header
-    background-color: #0072ff
+    background-color: #00979D
```

Comment:
Changing `#00979D` to `LIGHT_GREEN_HEADER` doesn't work, strangely? It seems that `app/layout/index.styl` doesn't automatically get the var values defined in `css/common.styl`